README.md

```diff
@@ -21,17 +22,12 @@ Create a new SearXNG instance in five minutes using Docker
   ```
 - Edit the [.env](https://github.com/searxng/searxng-docker/blob/master/.env) file to set the hostname and an email
 - Generate the secret key `sed -i "s|ultrasecretkey|$(openssl rand -hex 32)|g" searxng/settings.yml`
-- Edit the [searxng/settings.yml](https://github.com/searxng/searxng-docker/blob/master/searxng/settings.yml) file according to your need
-- Check everything is working: `docker compose up`
-- Run SearXNG in the background: `docker compose up -d`
+- Edit [searxng/settings.yml](https://github.com/searxng/searxng-docker/blob/master/searxng/settings.yml) according to your needs
```

Comment:
> The first few steps are the same for both installation methods however.

Aren't these steps not needed for both installation methods?

To be more clear what is the *first*, *second*, *third*, .. step we may better use a numbered list? ..

1. [Install docker](https://docs.docker.com/install/)
2. Get searxng-docker
3. Generate the secret key `sed -i "s|ultrasecretkey|$(openssl rand -hex 32)|g" searxng/settings.yml`
4. Edit the [.env](https://github.com/searxng/searxng-docker/blob/master/.env) file to set the hostname and an email
5. Edit [searxng/settings.yml](https://github.com/searxng/searxng-docker/blob/master/searxng/settings.yml) according to your needs

Reply:
Yeah, good idea :+1: